core/src/main/java/io/questdb/griffin/engine/functions/eq/EqLong256FunctionFactory.java

```diff
@@ -52,7 +52,7 @@ public Func(Function left, Function right) {
         @Override
         public boolean getBool(Record rec) {
             final Long256 lv = left.getLong256A(rec);
-            final Long256 rv = right.getLong256B(rec);
+            final Long256 rv = right.getLong256A(rec);
```

Comment:
A `Function` argument of the same class isn't a problem here and in other similar `Function`s because `left` and `right` will be different `Function` instances. Example SQL expression: `long256_col = long256_col`